code/game/objects/items/misc.dm

```diff
@@ -117,3 +115,64 @@
 	gender = PLURAL
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "ectoplasm"
+
+
+/obj/item/compactorebox
+	name = "compressed ore box"
+	desc = "A heavy and preety solid box, filled to the brim with ore, you probably shouldn't see it"
+	icon = 'icons/obj/items/items.dmi'
```

Comment:
Typo
Also you should be able to examine it while inhand so that comments unneded

Reply:
Fixed the typo , but  what do you mean by "you should be able to examine it while inhand" ?